sql/i4_invalid_indexes.sql

```diff
@@ -0,0 +1,25 @@
+-- Lisf of invalid Indexes 
+
+-- Use it to see invalid indexes list
+
+-- This query doesn't need any additional extensions to be installed
+-- (except plpgsql), and doesn't create anything (like views or smth)
+-- -- so feel free to use it in your clouds (Heroku, AWS RDS, etc)
+
+-- It also does't do anything except reading system catalogs and
+-- printing NOTICEs, so you can easily run it on your
+--  production *master* database.
+-- (Keep in mind, that on replicas, the whole picture of index usage
```

Comment:
This report is not about usage at all. It's "static" analysis -- similar to Redundant Indexes report.